ereolen_support/ereolen_support.helper.inc

```diff
@@ -387,8 +387,8 @@ function _ereolen_support_support_id_validate($element, $form_state) {
     if ($form_state['values']['ereolen_support_problem'] == $field['value']) {
       if ($element['#value'] != '') {
         // Assumes php mbstring extension enabled.
-        if (mb_strlen($element['#value']) != 5) {
-          form_set_error($element['#name'], $element['#title'].' should be 5 characters');
+        if (mb_strlen($element['#value']) != 6) {
```

Comment:
Why not user `drupal_strlen` https://api.drupal.org/api/drupal/includes!unicode.inc/function/drupal_strlen/7.x so that assumption is not needed?

Reply:
It the value "6" available as config some where?

Reply:
Changed. The length value is not a config and is assumed to be fairly static.